package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "bookstoreredux",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.9.7",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.1.3",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
```

Comment:
- [ ] When i tried running your code i noticed an error saying you do not have the react-router-dom package which you are making use of in your App.js file, you would need to install this so that your project works perfectly and can be used by anyone see the command below as a guide :+1: 
- Terminal command
```
npm i react-router-dom@latest
```